app/src/js/components/Operations/overview.js

```diff
@@ -164,15 +161,11 @@ class OperationOverview extends React.Component {
 
 OperationOverview.propTypes = {
   dispatch: PropTypes.func,
-  stats: PropTypes.object,
   operations: PropTypes.object,
-  collectionOptions: PropTypes.object,
-  workflowOptions: PropTypes.object
 };
 
 export default withRouter(connect(state => ({
   stats: state.stats,
```

Comment:
If `stats` is removed from the properties, should this be removed?

Reply:
Remove that as well as workflowOptions which doesn't seem to be used https://github.com/nasa/cumulus-dashboard/pull/797/commits/a828ef62e7cd0428cc1b151bcac3a71db4bbeca9